### PR TITLE
Update 1.22 comms team

### PR DESCRIPTION
**Pull request by savitharaghunathan**
#### What type of PR is this:

/kind documentation

#### What this PR does / why we need it:

Updating 1.22 comms team. Moving Jesse Butler to comms lead and adding an additional row to honor Peeyush

#### Which issue(s) this PR fixes:

None

#### Special notes for your reviewer:

None

/assign @annajung @divya-mohan0209 @erismaster @kikisdeliveryservice 
/cc @jeremyrickard @hasheddan @justaugustus @saschagrunert

### CODE REVIEW

File: releases/release-1.22/release-team.md
Comment (puerco) [inline releases/release-1.22/release-team.md:14]: :heart:

**Comment by savitharaghunathan** [comment]
> Thank you Savitha 🤗
Thank you @puerco <3 I procrastinated a bit on updating this doc and it's time now. This is the toughest one-liner PR I have ever made.

**Comment by savitharaghunathan** [comment]
/cc @puerco @cpanato 
Sorry, I wasn't thinking while adding the cc list. My apologies

**Comment by puerco** [comment]
We still have to finish our onboarding, but I can try :)
/approve

**Review by justaugustus — CHANGES_REQUESTED** [review]
@savitharaghunathan -- Thank you for memorializing Peeyush here!

Can we go a step further and add a link to his [CNCF memorial page](https://github.com/cncf/memorials/blob/main/peeyush-gupta.md), which includes a link to donate to the family?

**Comment by savitharaghunathan** [comment]
> @savitharaghunathan -- Thank you for memorializing Peeyush here!
> 
> Can we go a step further and add a link to his [CNCF memorial page](https://github.com/cncf/memorials/blob/main/peeyush-gupta.md), which includes a link to donate to the family?

Yes of course. Thank you @justaugustus. @mrbobbytables mentioned that as well. Update coming right away

**Comment by savitharaghunathan** [comment]
@justaugustus done. PTAL :) Let me know if should add it as a column instead of hyperlinking to the name

**Comment by justaugustus** [comment]
> @justaugustus done. PTAL :) Let me know if should add it as a column instead of hyperlinking to the name

Hmmmm, how about this:
- Pull this out of the table, under the Release Manager note in a separate `## In Memoriam` heading
- List the CNCF memorial page
- List the donation page

**Comment by savitharaghunathan** [comment]
@justaugustus Thanks for the super reviews. I moved it to a separate section at the end. Should I mention anything else?

**Review by saschagrunert — APPROVED** [review]
:heart: 
/lgtm
/hold
feel free to unhold if everything is fine

**Comment by cpanato** [comment]
/lgtm
<3

File: releases/release-1.22/release-team.md
Comment (justaugustus) [inline releases/release-1.22/release-team.md:21]:
```suggestion
- [Peeyush Gupta](https://github.com/cncf/memorials/blob/main/peeyush-gupta.md) (Emeritus Communications Lead)
  
  [Peeyush Gupta: Family Education Fund](https://milaap.org/fundraisers/support-peeyush-gupta-family-education)
```

**Review by justaugustus — APPROVED** [review]
@savitharaghunathan -- Committed a tiny tweak to formatting and called out the Communications role as well.

Thank you again for adding this!
/lgtm
/approve

**Pull request merged**
